tests/data/agb_live_woody_density/Aboveground_Live_Woody_Biomass_Density.geojson

```diff
@@ -1 +1,32 @@
-{"type": "FeatureCollection", "name": "Aboveground_Live_Woody_Biomass_Density", "crs": {"type": "name", "properties": {"name": "urn:ogc:def:crs:OGC:1.3:CRS84"}}, "features": [{"type": "Feature", "properties": {"tile_id": "00N_000E", "Mg_px_1_download": "tests/data/agb_live_woody_density/00N_000E.tif", "ObjectId": 1, "Shape__Area": 1245542622548.87, "Shape__Length": 4464169.76558139}, "geometry": {"type": "Polygon", "coordinates": [[[0.0, 0.0], [10.0, 0.0], [10.0, -10.0], [0.0, -10.0], [0.0, 0.0]]]}}]}
```

Comment:
These files are auto-generated. If we update these in this PR, running `data.py` again will change the file. Is there an option to ignore all `tests/data/*` directories?

Reply:
Yes we can specify in `.prettierignore` what we would like to ignore.